bettercli/option.py

```
@@ -1,88 +1,110 @@
+class ValidType[**P, R](t.Protocol):
+    def __init__(self, *args: P.args, **kwargs: P.kwargs) -> 'None': ...
+
+T = t.TypeVar("T", bound=ValidType)
 
-class Keyword_option(Option[T]):
-    def __init__(self, name:'str', type=None, *keys, default=None, length=None, min_length=None, max_length=None):
-        logger.debug(f"Keyword_option.__init__: {name=} {type=} {keys=} {default=} {length=} {min_length=} {max_length=}")
+
+
+class Keyword_option(t.Generic[T]):
+    def __init__(self, name:'str', type:'list[type[T]]'=[None], *keys, default:'list[T]'=[None], length=1, max_occurrences=1):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)